driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -182,6 +188,23 @@ public RequestHandler(
     } else if (statement instanceof BoundStatement) {
       statementType = "prepared";
       statementText = ((BoundStatement) statement).statement.getQueryString();
+      keyspace = ((BoundStatement) statement).getKeyspace();
+
+      ColumnDefinitions boundColumns =
+          ((BoundStatement) statement).statement.getPreparedId().boundValuesMetadata.variables;
+
+      StringBuilder partitionKeyBuilder = new StringBuilder(PARTITION_KEY_MAX_LENGTH);
+      for (int i = 0; i < boundColumns.size(); i++) {
+        String columnName = boundColumns.getName(i);
+        Object value = ((BoundStatement) statement).getObject(i);
+        String valueString = value.toString();
```

Comment:
When `value` is `null`, this crashes with `NullPointerException` as shown in one of integration tests:
```
prepareWithNullValuesTest(com.datastax.driver.core.PreparedStatementTest)  Time elapsed: 0.004 sec  <<< FAILURE!
java.lang.NullPointerException: null
	at com.datastax.driver.core.RequestHandler.<init>(RequestHandler.java:200)
	at com.datastax.driver.core.SessionManager.execute(SessionManager.java:734)
	at com.datastax.driver.core.SessionManager.execute(SessionManager.java:750)
	at com.datastax.driver.core.SessionManager.executeAsync(SessionManager.java:163)
	at com.datastax.driver.core.AbstractSession.execute(AbstractSession.java:58)
	at com.datastax.driver.core.PreparedStatementTest.prepareWithNullValuesTest(PreparedStatementTest.java:427)
```